src/library-authoring/components/ComponentCard.tsx

```diff
@@ -50,14 +53,22 @@ export const ComponentMenu = ({ usageKey }: { usageKey: string }) => {
         data-testid="component-card-menu-toggle"
       />
       <Dropdown.Menu>
-        <Dropdown.Item disabled>
-          {intl.formatMessage(messages.menuEdit)}
-        </Dropdown.Item>
+        {
+          editUrl ? (
+            <Dropdown.Item as={Link} to={editUrl}>
+              <FormattedMessage {...messages.menuEdit} />
+            </Dropdown.Item>
+          ) : (
+            <Dropdown.Item disabled>
+              <FormattedMessage {...messages.menuEdit} />
+            </Dropdown.Item>
+          )
+        }
```

Comment:
Same as above
```suggestion
        <Dropdown.Item disabled={!editUrl} as={Link} to={editUrl}>
          <FormattedMessage {...messages.menuEdit} />
         </Dropdown.Item>
```


Not related to this task, but we have two edit options in the sidebar:
![image](https://github.com/user-attachments/assets/9ecf4446-9cb2-4995-a042-9464c9d90f12)

Reply:
Fixed in cdd2b63f70c4dfa47faa9396ecdd8df62567a446 . And yeah, it's kinda weird having two edit buttons so close together.